src/python/gudhi/hera/wasserstein.cc

```diff
@@ -8,29 +8,126 @@
  *      - YYYY/MM Author: Description of the modification
  */
 
-#include <wasserstein.h> // Hera
-
 #include <pybind11_diagram_utils.h>
 
-double wasserstein_distance(
+#ifdef _MSC_VER
+// https://github.com/grey-narn/hera/issues/3
+// ssize_t is a non-standard type (well, posix)
+using py::ssize_t;
+#endif
```

Comment:
This hack is there because of a bug in Hera, I'd rather have it only in files that use Hera.
As I mentioned, it could make sense to merge the 2 cc files in hera, now that Hera can handle including bottleneck and wasserstein in the same translation unit. Then this part would only appear once...